web/src/datavzrd.js

```diff
@@ -68,6 +68,7 @@ function renderTickPlot(ah, columns, title, slug_title, specs, is_float, precisi
     if (detail_mode || header_label_length !== 0) {
         index += 1;
     }
+    index += 1; // compensate for line numbers
```

Comment:
This looks repetetive and easy to forget

Reply:
I changed the index when initializing it. Looks cleaner now thanks!